package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "start": "tsdx watch",
     "build": "tsdx build",
+    "build:watch": "tsdx watch --onSuccess \"yalc push\"",
```

Comment:
Added this so we can automatically push to yalc after a successful build

Reply:
should we add yalc as a dev dependency then?

Reply:
I think yalc is added as a global dependency, it's already in the docs. Not sure if we should add as a dev, or just leave the script for anyone using yalc

Reply:
right, should have installed yalc globally. No action needed here then.